packages/special-pages/pages/onboarding/app/components/Typed.js

```diff
@@ -14,13 +15,29 @@ import { SettingsContext } from '../settings'
  * @param {number} [props.delay=20] - The delay (in milliseconds) between each character being typed.
  */
 export function Typed ({ text, children = null, onComplete = null, delay = 20, ...rest }) {
+    const globalState = useContext(GlobalContext)
+    const { activeStep } = globalState
+    const pre = useRef(/** @type {string|undefined} */(undefined))
+    useEffect(() => {
+        if (activeStep && pre.current) {
+            if (text === pre.current) {
+                onComplete?.()
+                return
+            }
+        }
+        pre.current = text
+    }, [activeStep, text])
     return (
-        <TypedInner key={text} text={text} onComplete={onComplete} delay={delay} {...rest}>{children}</TypedInner>
+        <TypedInner key={text}
+            text={text}
+            onComplete={onComplete}
+            delay={delay}
+            {...rest}>{children}</TypedInner>
```

Comment:
I couldn't think of a better way to solve this problem, without a bigger rewrite. 

In short, this is looking if the current page title matches the old one - if it does, we just call the callback (which is used by the logic that handles advancing screens).

This is certainly something to consider improving later, but works for now.